benchmarks/util.py

```diff
@@ -1,154 +1,155 @@
-from contextlib import contextmanager
 import functools
 import logging
-from multiprocessing import Process, Queue
-import numpy as np
 import os
-from os.path import abspath
 import queue
 import random
 import subprocess
-import torch
 import traceback
+from contextlib import contextmanager
+from multiprocessing import Process, Queue
+from os.path import abspath
+
+import numpy as np
+import torch
 
 try:
-  import torch_xla.core.xla_model as xm
-  from torch_xla.distributed.cluster import ClusterResolver
+    import torch_xla.core.xla_model as xm
+    from torch_xla._internal import tpu
 except ImportError:
-  # ignore the error if torch_xla is not installed
-  pass
+    # ignore the error if torch_xla is not installed
+    pass
 
 logger = logging.getLogger(__name__)
 
 
 @functools.lru_cache(None)
 def patch_torch_manual_seed():
-  """Make torch manual seed deterministic. Helps with accuracy testing."""
+    """Make torch manual seed deterministic. Helps with accuracy testing."""
 
-  def deterministic_torch_manual_seed(*args, **kwargs):
-    from torch._C import default_generator
+    def deterministic_torch_manual_seed(*args, **kwargs):
+        from torch._C import default_generator
 
-    seed = 1337
-    import torch.cuda
+        seed = 1337
+        import torch.cuda
 
-    if not torch.cuda._is_in_bad_fork():
-      torch.cuda.manual_seed_all(seed)
-    return default_generator.manual_seed(seed)
+        if not torch.cuda._is_in_bad_fork():
+            torch.cuda.manual_seed_all(seed)
+        return default_generator.manual_seed(seed)
 
-  torch.manual_seed = deterministic_torch_manual_seed
+    torch.manual_seed = deterministic_torch_manual_seed
 
 
 def reset_rng_state(benchmark_experiment=None):
-  torch.manual_seed(1337)
-  random.seed(1337)
-  np.random.seed(1337)
-  if benchmark_experiment and benchmark_experiment.xla:
-    device = benchmark_experiment.get_device()
-    xm.set_rng_state(1337, str(device))
+    torch.manual_seed(1337)
+    random.seed(1337)
+    np.random.seed(1337)
+    if benchmark_experiment and benchmark_experiment.xla:
+        device = benchmark_experiment.get_device()
+        xm.set_rng_state(1337, str(device))
 
 
 @functools.lru_cache(maxsize=3)
 def is_xla_device_available(devkind):
-  if devkind not in ["CPU", "CUDA", "TPU"]:
-    raise ValueError(devkind)
+    if devkind not in ["CPU", "CUDA", "TPU"]:
```

Comment:
any reason to change the indentation?
If all the other files indents 2 spaces, let's be consistent and also indent 2 spaces

Reply:
bad merge